adapters/Nend/NendAdapter/GADMAdapterNendAdUnitMapper.m

```diff
@@ -0,0 +1,25 @@
+//
+//  GADMAdapterNendAdUnitMapper.m
+//  NendAdapter
+//
+//  Copyright © 2019 FAN Communications. All rights reserved.
+//
+
+#import "GADMAdapterNendAdUnitMapper.h"
+
+@implementation GADMAdapterNendAdUnitMapper
+
++ (BOOL)validateApiKey:(NSString *)apiKey spotId:(NSString *)spotId {
+    if (!apiKey || apiKey.length == 0 || !spotId || spotId.length == 0) {
+        return false;
+    }
+    return true;
+}
+
++ (NSString *)mappingAdUnitId:(id<GADMAdNetworkConnector>)connector
```

Comment:
Is this helper method necessary since theres no validation of any kind? I think we can just directly get the `paramKey` from the `connector` without the helper method.

Reply:
It changed by our internal reason.

We would like to commonize all process that getting SPOT ID and API KEY on subclass of `GADMAdNetworkAdapter` near future (but NOT this PR)
- `mappingAdUnitId`
- `getNendAdParam`

Reply:
This is acknowledged.